packages/sources/finnhub/src/endpoint/quote.ts

```diff
@@ -70,6 +72,10 @@ export const buildSymbol = ({
   quote,
   exchange,
 }: typeof inputParameters.validated): string => {
+  if (isExchangeSymbol(base)) {
+    return base
+  }
```

Comment:
Out of scope for this PR but I want to get your thoughts on potentially removing exchange data from `overrides` and instead passing it as a separate param in `includes`? This would allow us to avoid matching regex in the request.

Reply:
Resolving thread so I can merge, but I'm definitely keen to find neater ways to structure this!

Could you provide an example of what that might look like? I'm not quite following right now